src/util/mw-api.ts

```diff
@@ -253,3 +254,12 @@ export function mwRetToArticleDetail(obj: QueryMwRet): KVS<ArticleDetail> {
   }
   return ret
 }
+
```

Comment:
Maybe a comment about what the purpose of this? At a first look, I can not remember.

Reply:
I replaced checkApiAvailabilty() method from Downloader because it is more like a utility function.